src/cascadia/TerminalSettingsEditor/EnumEntry.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "EnumEntry.h"
+#include "EnumEntry.g.cpp"
+
+using namespace winrt;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Microsoft::Terminal::Settings::Model;
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    EnumEntry::EnumEntry(const winrt::hstring enumName, const winrt::Windows::Foundation::IInspectable& enumValue)
```

Comment:
nit: prefer using constructor syntax (`EnumValue(...): _x{x}, _y{y} { }` instead of calling our local setter functions. We don't need to dispatch observable notifications while constructing, so there's nothing to be gained by calling them except complexity

Reply:
oh that and i probably could just do away with the `.cpp` if it's only gonna have a ctor that i can put in the `.h`?

Reply:
bingo!